evm/chains/sharding/mainchain_handler/geth_shell_scripts/launch_geth.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
I question the inclusion of this file/script. Why is it present?  Why are we using geth?  I was somewhat assuming that we'd be interfacing with py-evm for this?

Mostly just looking for an explanation so that I can better understand the context of including these files.

Reply:
You're right. I should not put the shell scripts here. I will remove them.
The idea is
> (ii) a "shard client" which talks to the main chain node via RPC (assumed to be trusted because it's also running on the user's computer)
in https://github.com/ethereum/sharding/blob/develop/docs/doc.md#introduction
So I should not limit the code to use only geth.